app/controllers/api/v1/places_controller.rb

```diff
@@ -0,0 +1,6 @@
+require 'pry'
```

Comment:
Please remove! (You shouldn't need to require pry anyways, but this line definitely shouldn't be committed)